crates/astria-conductor/justfile

```diff
@@ -0,0 +1,18 @@
+alias r := run
```

Comment:
Nit: I wouldn't complicate it with extra abbreviations.

Reply:
I don't feel strongly about this at all, but isn't it a very common pattern to alias most used commands?

Reply:
Well, "common" here is probably by less than 10 people and only 3 times.

It's IMO more confusing to have abbreviations for very rarely used tools